grassroots-backend/src/phone-canvass/PhoneCanvass.service.ts

```diff
@@ -120,4 +125,61 @@ export class PhoneCanvassService {
       },
     });
   }
+
+  async updateSyncData(phoneCanvassId: string): Promise<void> {
+    const contacts = await this.getPhoneCanvassContacts(phoneCanvassId);
+
+    const partitionedContacts = partition(contacts, (contact) => {
+      if (contact.callStatus === "NOT_STARTED") {
+        return "NOT_STARTED";
+      } else if (contact.callStatus === "STARTED") {
+        return "STARTED";
+      }
+      return "COMPLETE";
+    });
+
+    const activeCalls: ActiveCall[] = (
```

Comment:
These bits are still untested at this point. My next step is probably to start writing some tests which simulate actually making calls, which will cover this.